core/put/source/dev_ui.cpp

```diff
@@ -342,6 +342,16 @@ namespace put
 
             create_render_states();
 
+            
+            style.ScaleAllSizes(DEV_UI_SCALE);
+
+            dev_ui::util_init();
+            s_initialised = true;
+
+            return true;
+        }
+        
+        ImGuiStyle &default_pmtech_style(){
```

Comment:
Please follow convention for references to attach to the typename and Allan braces
```
ImGuiStyle& default_pmtech_style
{
```